src/test/http_rest_api_handler_test.cpp

```diff
@@ -56,6 +56,12 @@ class ConfigApi : public TestWithTempDir {
     void RemoveConfig() {
         std::filesystem::remove(configFilePath);
     }
+
+    void SetUpSingleModel(std::string modelPath, std::string modelName) {
+        char* n_argv[] = {"ovms", "--model_path", &modelPath[0], "--model_name", &modelName[0], "--file_system_poll_wait_seconds", "0"};
```

Comment:
modelPath.data() or modelPath.c_str() to get pointer of underlying char array

Reply:
Are you sure that buffers do not need to exist after parsing? Lifetime of modelPath/modelName ends at the end oof SetUpSingleModel function